packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -194,11 +147,11 @@ export class SwapQuoter {
             }
         });
 
-        const opts = { ...constants.DEFAULT_SWAP_QUOTE_REQUEST_OPTS, ...options };
+        const _opts = { ...constants.DEFAULT_SWAP_QUOTE_REQUEST_OPTS, ...opts };
```

Comment:
I know this is a preexisting condition but could we name this better? `opts` and `_opts` is very confusing. Maybe `optsWithDefaults` or smth?